src/main/java/Duke.java

```diff
@@ -15,19 +19,59 @@ public static void main(String[] args) {
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
 
+        try {
+            File f = new File("./data/duke.txt");
+            Scanner scannerOne = new Scanner(f);
+            while(scannerOne.hasNextLine()) {
+                String[] parts = scannerOne.nextLine().split(Pattern.quote(" | "));
+                switch (parts[0]) {
+                    case "T":
```

Comment:
For tasks, maybe consider using Factory methods inside a parser. You can then encapsulate the parsing logic inside the parser. With that said, this may result in 
```java
case "D":
case "T":
case "E":
   parser.add(response);
   break;
```
Just a suggestion to keep code concise, but currently still looks good 👍👍. For your consideration.